core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
```

Comment:
@seanlip . I can make that change. I have a small query though. We don't need to move the function to exp_services (or exp_domain), do we?